.github/workflows/code-quality.yml

```
@@ -0,0 +1,212 @@
+            echo "✅ Branded types properly defined"
+          fi
+      
+      - name: Check React patterns
+        run: |
+          echo "🔍 Checking React patterns..."
+          
+          # Check for proper hook usage
+          if grep -r "use[A-Z]" src/hooks/ --include="*.ts" | grep -v "use[A-Z][a-zA-Z]*"; then
+            echo "✅ Custom hooks follow naming convention"
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)